site/docs/basics/aop.zh-CN.md

```
@@ -0,0 +1,346 @@
+根据上面的实现过程切面函数可以通过`AdviceContext`来影响被切函数的执行：
+
+```typescript
+@Advice()
+class PointcutAdvice implements IAdvice<Hello> {
+
+  @Inject()
+  logger: EggLogger;
+
+  // 修改被切函数的入参
+  async beforeCall(ctx: AdviceContext<Hello>): Promise<void> {
+  	ctx.args = [ 'for', 'bar' ];
```

markdownlint-cli2 (0.18.1)

Hard tabs
Column: 3

(MD010, no-hard-tabs)